index.html

```diff
@@ -4,25 +4,25 @@
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 	<link rel="stylesheet" href="assets/css/dist/style.css" />
 	<link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900" rel="stylesheet">
+	<link rel="shortcut icon" href="assets/img/favicon.ico" type="image/x-icon">
+	<link rel="icon" href="assets/img/favicon.ico" type="image/x-icon">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Blox - Open Source Tools for Amazon ECS</title>
+	<title>Blox - Scheduler for Amazon ECS</title>
 </head>
 <body>
-
+	
 	<div id="top" class="header">
 		<div class="container">
 			<div class="row">
-				<div class="col-md-9">
-					<p class="lead"><img style="height: 40px" src="assets/img/blox-logo.svg" class="blox" alt="Blox" /> is a collection of open source projects
-					for container management and orchestration on <a href="https://aws.amazon.com/ecs">Amazon ECS</a>
+				<div class="col-md-7">
+					<p class="lead"><img style="height: 40px" src="assets/img/blox-logo.svg" class="blox" alt="Blox" />&nbsp;&nbsp;is an open source scheduler built for <a href="https://aws.amazon.com/ecs">Amazon EC2 Container Service</a>
 					</p>
 				</div>
 			</div>
 
 			<div class="row">
-				<div class="col-sm-9">
-					<p class="block-description sublead">Blox gives you more control over how your containerized applications run on Amazon ECS.
-						It enables you to build schedulers and integrate third-party schedulers on top of ECS, while leveraging Amazon ECS to fully manage and scale your clusters.</p>
+				<div class="col-sm-9 tab">
+					<p class="block-description sublead">Blox provides scheduling capabilities that enable developers to have greater choice and flexibility over when to deploy, run, and scale tasks for containerized services.</p>
```

Comment:
I am not sure this copy is correct. I don't understand what "greater choice" and "flexibility" means? What are we trying to get at? Its sort of a nebulous sentence.